.github/workflows/build-build-tools-image.yml

```
@@ -107,8 +124,11 @@ jobs:
           username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
           password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
 
-      - name: Create multi-arch image
+      - name: Create multi-arch tool image
```

AFAIR we can use `${{ matrix.tool }}` here and the value will be correctly interpolated 

```suggestion
      - name: Create multi-arch ${{ matrix.tool }} image
```